ui/components/component-library/button/button.stories.tsx

```diff
@@ -122,10 +122,14 @@ export const SizeStory: StoryFn<typeof Button> = (args) => (
 SizeStory.storyName = 'Size';
 
 export const Danger: StoryFn<typeof Button> = (args) => (
-  <Box display={Display.Flex} gap={1}>
-    <Button {...args}>Normal</Button>
-    {/* Test Anchor tag to match exactly as button */}
-    <Button as="a" {...args} href="#" danger>
+  <Box display={Display.Flex} gap={4}>
+    <Button danger>
+      Danger
+    </Button>
+    <Button variant={ButtonVariant.Secondary} danger>
+      Danger
+    </Button>
+    <Button {...args} variant={ButtonVariant.Link} danger>
       Danger
     </Button>
```

Comment:
Updating story to show all danger variants 

<img width="333" alt="Screenshot 2025-06-23 at 12 46 24 PM" src="https://github.com/user-attachments/assets/7c457ce8-c353-436f-9d2e-fa4ba1dc1ea1" />